docs/howtos/releases.md

```diff
@@ -0,0 +1,99 @@
+# Application Services Release Process
+
+## Nightly builds
+
+Nightly builds are automatically generated using a taskcluster cron task.
+
+- The results of the latest successful nightly build is listed here:
+  https://firefox-ci-tc.services.mozilla.com/tasks/index/project.application-services.v2.nightly/latest
+- The latest nightly decision task should be listed here:
+  https://firefox-ci-tc.services.mozilla.com/tasks/index/project.application-services.v2.branch.main.latest.taskgraph/decision-nightly
+- If you don't see a decision task from the day before, then contact releng.  It's likely that the cron decision task is broken.
+
+## Release builds
+
+Release builds are generated from the `release-vXXX` branches.  Whenever a commit is pushed there,
+we build all artifacts needed for the release.  Once we're ready to publish the release, we run the
+taskcluster release promotion action which signs and publishes the artifacts.
+
+TODO: explain release-promotion more.
+
+## What to do at the end of a nightly cycle
+
+1. Start this 2 workdays before the nightly cycle ends
```

Comment:
I just made up this 2 workdays number, not sure when we should start it.  We should make sure we have plenty of time to finish before firefox-android or firefox-ios want to start cutting their releases.